uz.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="uz">
+<head>
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-141126533-1"></script>
+    <script>
+        window.dataLayer = window.dataLayer || [];
+
+        function gtag() {
+            dataLayer.push(arguments);
+        }
+
+        gtag('js', new Date());
+        gtag('config', 'UA-141126533-1');
+    </script>
+    <meta charset="utf-8">
+    <title>nometa</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" media="screen" href="./main.css">
+<body>
+<article>
+    <div class="internationalisation">
+        <a href="index.html">English version</a>
+        <a href="uk.html">Українська версія</a>
+        <a href="ru.html">Русская версия</a>
+    </div>
+    <h1>Iltimos, chatda meta-savollar bermang!</h1>
+
+    <p><b>Meta-savol</b> — bu boshqa savollarni nazarda tutuvchi savol, masalan:</p>
+
+    <div class="dialog-line -you">
+        <div class="content">
+            Savol bersam maylimi?
+        </div>
+    </div>
+
+    <div class="dialog-line -you">
+        <div class="content">
+            N ga tushunadiganlar bormi?
+        </div>
+    </div>
+
+    <div class="dialog-line -you">
+        <div class="content">
+            Bu yerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bilan oid savollar bor...
```

Comment:
```suggestion
            Bu yerda kimdir Yandexdan kurs sotib olganmi? Menda matn tahlili bo'yicha savollar bor...
```
@thetakhirov 
These words are not correct.
This method is easy to read and to understand

Reply:
Thanks 😺